src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -425,7 +466,10 @@ internal static SecurityKey ResolveTokenSigningKey(string kid, string x5t, Token
                 }
             }
 
-            return null;
+            if (validationParameters.Configuration != null)
```

Comment:
Not sure if it matters to do this check first, if one of the keys in validationParameters.IssuerSigningKeys works does it impact the decision of saving/using LKG?

Reply:
I agree that the check doesn't really matter, we could execute ResolveTokenSigningKeyWithConfig regardless of whether the configuration is present or not. But if it's null we avoid an additional method call.

The LKG will be saved even if the TVP was responsible for successful validation.

Reply:
@mafurman i agree with the null check as for any current asp.net users Configuration will always be null.